### PR TITLE
Remove protocol-relative URLs

### PR DESCRIPTION
**Pull request by konklone**
Removes ([now-deprecated](http://www.paulirish.com/2010/the-protocol-relative-url/)) protocol-relative URLs in favor of https:// URLs.

### CODE REVIEW

File: entries2html.xsl [hunk 1]
```diff
@@ -12,7 +12,7 @@
 	&lt;meta charset="utf-8"&gt;
 	&lt;title&gt;<xsl:value-of select="//entry/@name"/> demo&lt;/title&gt;<xsl:if test="css">
 	&lt;style&gt;<xsl:value-of select="css/text()"/>	&lt;/style&gt;</xsl:if>
-	&lt;script src="//code.jquery.com/jquery-1.10.2.js"&gt;&lt;/script&gt;<xsl:if test="code/@location='head'">
+	&lt;script src="https://code.jquery.com/jquery-1.10.2.js"&gt;&lt;/script&gt;<xsl:if test="code/@location='head'">
```
Comment (gnarf) [inline entries2html.xsl]: Should we bump to 1.11.2 while we are at it?

Comment (arthurvr) [inline entries2html.xsl]: Yes, same thing in the PR for the learn site.

Comment (AurelioDeRosa) [inline entries2html.xsl]: Yes, I agree with @gnarf.

Comment (arthurvr) [inline entries2html.xsl]: We discussed this in the meeting feb 11, and decided to do this separately. https://github.com/jquery/api.jquery.com/issues/651

**Pull request closed without merging**